.circleci/config.templ.yml

```diff
@@ -652,15 +652,6 @@ jobs:
           snapshot: true
           docker_services: "memcached redis postgres"
 
-  djangorestframework:
```

Comment:
FYI this is also done in: https://github.com/DataDog/dd-trace-py/pull/10611/files